x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/rollup/ConfigTestHelpers.java

```diff
@@ -141,4 +139,8 @@ public static TermsGroupConfig randomTermsGroupConfig(final Random random) {
     private static String randomField(final Random random) {
         return randomAsciiAlphanumOfLengthBetween(random, 5, 10);
     }
+
+    private static long randomInterval(final Random random) {
+        return RandomNumbers.randomLongBetween(random, 0L, Long.MAX_VALUE);
```

Comment:
I think this needs to be `randomLongBetween(random, 1L, Long.MAX_VALUE);`, since the validation check makes sure the interval is a value greater than 0.

Reply:
Good catch